scripts/macros/heal.js

```diff
@@ -0,0 +1,60 @@
+const healTokens = async () => {
+  if (!game.clt?.supported) {
+    console.error(
+      'To use the Heal macro, you should install Condition Lab & Triggler'
+    );
+  }
+
+  const selectedTokens = canvas.tokens.controlled;
+
+  selectedTokens.forEach(async (token) => {
+    const tokenActor = token.actor;
+    if (!['character', 'mook'].includes(tokenActor.type)) {
+      return;
+    }
+
+    const name = tokenActor.type === 'character' ? tokenActor.name : token.name;
+    const body = tokenActor.system.stats.body.value;
+    let baseHeal = tokenActor.system.stats.body.value;
+    let healedAmtString = `<ul><li>${baseHeal} [BODY]</li>`;
+
+    // Enhanced Antibodies Cyberware (x2) natural healing rate
+    if (tokenActor.itemTypes.cyberware.find((c) => c.name.match(/antibodies/i))) {
```

Comment:
careful here, not everyone is playing in english. Some people might play in another language and use babel for auto translated items. Here one idea would be to check if we could use babels translations as they are readily available from the core system